app/assets/stylesheets/style.scss

```diff
@@ -0,0 +1,154 @@
+* {
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+// main,
+// header,
+// footer {
+//   display: flex;
+//   flex-direction: column;
+//   align-items: center;
+// }
```

Comment:
- [ ] Following the best practices please remove this unused commented code from the codebase